scripts/mdpow-solvationenergy

```diff
@@ -105,20 +105,59 @@ def run_gsolv(solvent, directory, **kwargs):
 
     gsolv = load_gsolv(solvent, directory, permissive=kwargs.pop('permissive',False),)
 
+    if not hasattr(gsolv, 'start'):
+        gsolv.start = kwargs.pop('start', 0)
+    if not hasattr(gsolv, 'stop'):
+        gsolv.stop = kwargs.pop('stop', None)
```

Comment:
add a INFO logger that states the start and stop values that are going to be used